src/main/java/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,57 @@
+package domain.lotto;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import domain.lotto.number.LottoNumber;
+
+public class Lotto {
+	public static final int LOTTO_SIZE = 6;
+
+	private final Set<LottoNumber> lottoNumbers;
+
+	public Lotto(List<LottoNumber> lottoNumbers) {
+		Objects.requireNonNull(lottoNumbers);
+
+		validateSize(lottoNumbers);
+		this.lottoNumbers = new HashSet<>(lottoNumbers);
+		validateDuplicate();
+	}
+
+	private void validateDuplicate() {
+		if (this.lottoNumbers.size() != LOTTO_SIZE) {
+			throw new IllegalArgumentException("로또 번호는 중복될 수 없습니다." + this.lottoNumbers);
+		}
+	}
+
+	private void validateSize(List<LottoNumber> lottoNumbers) {
+		if (lottoNumbers.size() != LOTTO_SIZE) {
+			throw new IllegalArgumentException("로또는 로또넘버 6개로 만들 수 있습니다. lottoNumbers=" + lottoNumbers);
+		}
+	}
+
+	public boolean isContains(LottoNumber lottoNumber) {
```

Comment:
```suggestion
	public boolean contains(LottoNumber lottoNumber) {
```

동사 `contains`만으로도 충분히 의미하고자 하는 바를 표현할 수 있을 것 같아요.